packages/webapp/pages/onboarding.tsx

```diff
@@ -358,11 +386,16 @@ export function OnboardPage(): ReactElement {
             )}
             {activeScreen === OnboardingStep.ContentTypes && <ContentTypes />}
             {activeScreen === OnboardingStep.Sources && <Sources />}
+            {activeScreen === OnboardingStep.AndroidApp && (
+              <OnboardingAndroidApp />
+            )}
           </div>
         )}
       </div>
       {showOnboardingPage && <OnboardingFooter />}
-      <FooterLinks className="mx-auto pb-6" />
+      {activeScreen !== OnboardingStep.AndroidApp && (
```

Comment:
Maybe worth checking with design if it's intentional?
Just mention every step now includes it.